@kiva/kv-components/src/vue/KvLendCta.vue

```diff
@@ -519,9 +532,12 @@ export default {
 				|| this.isAmountBetween25And500(this.unreservedAmount));
 		},
 		isLendAmountButton() {
-			return ((this.lendButtonVisibility || this.state === 'lent-to')
+			if (!this.maxAmount) {
+				return ((this.lendButtonVisibility || this.state === 'lent-to')
 				&& this.isAmountLessThan25(this.unreservedAmount))
 				|| (this.showPresetAmounts && this.presetButtonsPrices.length === 1 && !this.isAdding);
+			}
+			return false;
```

Comment:
Why do these two need to return `false` when a max amount is used?

Reply:
We want the behavior shown in my screenshots, if these 2 arent manually set to false, we get a lend button displayed: "Lend $<UnreservedAmount>"  for the cases when maxAmount is only enough for 1 button (maxAmount: '25')